docker/docker-compose.test.yml

```diff
@@ -17,6 +17,7 @@ services:
 
   timescale:
     image: timescale/timescaledb:latest-pg12
+    command: postgres -F
```

Comment:
Cute. Can you actually tell it being faster?

Reply:
yes, when I first did this with AB you could visibly see the tests running faster.